twistlock/datadog_checks/twistlock/twistlock.py

```diff
@@ -0,0 +1,322 @@
+                "aggregation_key": cve_id,
+                'host': self.hostname
+            }
+
+            self.event(event)
+
+    def _report_vuln_info(self, namespace, data, tags):
+        # CVE vulnerabilities
+        summary = Counter({"critical": 0, "high": 0, "medium": 0, "low": 0})
+        cves = data.get('info', {}).get('cveVulnerabilities', []) or []
+        for cve in cves:
+            summary[cve['severity']] += 1
+            cve_tags = [
+                'cve:' + cve['cve'],
+            ] + SEVERITY_TAGS.get(cve['severity'], []) + tags
+            if 'packageName' in cve:
+                cve_tags += ["package:" + cve['packageName']]
+            self.gauge(namespace + '.cve.details', float(1), cve_tags)
+        # Send counts to avoid no-data on zeroes
+        for severity, count in iteritems(summary):
+            cve_tags = SEVERITY_TAGS.get(severity, []) + tags
+            self.gauge(namespace + '.cve.count', float(count), cve_tags)
+
+    def _report_compliance_information(self, namespace, data, tags):
+        compliance = Counter({"critical": 0, "high": 0, "medium": 0, "low": 0})
+        vulns = data.get('info', {}).get('complianceDistribution', {}) or {}
+        types = ["critical", "high", "medium", "low"]
+        for type in types:
+            compliance[type] += vulns[type]
+            compliance_tags = SEVERITY_TAGS.get(type, []) + tags
+            self.gauge(namespace + '.compliance.count', compliance[type], compliance_tags)
+
+    def _report_layer_count(self, data, namespace, tags):
+        # Layer count and size
+        layer_count = 0
+        layer_sizes = 0
+        for layer in data.get('info', {}).get('history', []):
+            layer_count += 1
+            layer_sizes += layer.get('sizeBytes', 0)
+        self.gauge(namespace + '.size', float(layer_sizes), tags)
+        self.gauge(namespace + '.layer_count', float(layer_count), tags)
+
+    def _report_service_check(self, data, prefix, format, tags=[], message=""):
+        # Last scan service check
+        scan_date = datetime.strptime(data.get("scanTime"), format)
+        scan_status = AgentCheck.OK
+        if scan_date < self.warning_date:
+            scan_status = AgentCheck.WARNING
+        if scan_date < self.critical_date:
+            scan_status = AgentCheck.CRITICAL
+        self.service_check(prefix + '.is_scanned',
+                           scan_status,
+                           tags=tags,
+                           message=message)
+
+    def _retrieve_json(self, path, page=0):
+        url = self.config.url + path
+        auth = (self.config.username, self.config.password)
+        response = requests.get(url, auth=auth, verify=self.config.ssl_verify)
+        try:
+            j = response.json()
+            # it's possible to get a null response from the server
+            # {} is a bit easier to deal with
+            if not j:
+                return {}
+            return j
```

Comment:
Instead of if branch maybe
```suggestion
            return j or {}
```